app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -193,6 +234,7 @@ class BrowserToolbarIntegration(
     @VisibleForTesting
     internal fun stopObserverSecurityIndicatorChanges() {
         securityIndicatorScope?.cancel()
+        tabsCounterScope?.cancel()
```

Comment:
nit: This should probably happen in `stop()` directly and not in `stopObserverSecurityIndicatorChanges()` which is unrelated.